scripts/apps/master-desk/MasterDesk.tsx

```diff
@@ -113,6 +113,13 @@ export class MasterDesk extends React.Component<{}, IState> {
         return this.state.currentTab === IMasterDeskTab.overview;
     }
 
+    onTabChange(tab) {
```

Comment:
all custom methods should be bound in the constructor

Reply:
done